src/apps/reducers.js

```diff
@@ -0,0 +1,59 @@
+import { UPDATE_LOCATION } from 'react-router-redux';
+import { combineReducers } from 'redux';
+import { NEW_BUNDLE, NEW_ERROR, FETCH_BUNDLE_ERROR } from './actions';
+
+function runMode(state = { path: null, rendezvous: null }, action) {
+    if (action.type === UPDATE_LOCATION) {
+        let { query } = action.payload;
+
+        if (query.path) {
```

Comment:
possibly better to write as

```javascript
return {
  path: query.path || null,
  rendezvous: query.rendezvous || null
}
```

or if `query` can only contain `path` or `rendezvous`

```javascript
return Object.assign({path: null, rendezvous: null}, query);
```